apis/config/v1beta1/configuration_types.go

```diff
@@ -321,7 +321,8 @@ type Integrations struct {
 	//  - "kubeflow.org/tfjob"
 	//  - "kubeflow.org/xgboostjob"
 	//  - "pod"
-	//  - "deployment"
+	//  - "deployment" (requires enabling pod integration)
+	//  - "statefulset" (requires enabling pod integration)
```

Comment:
Could you also update https://github.com/kubernetes-sigs/kueue/blob/7dfe92170cfd159d305a9b46abd7b4e2a2338b61/config/components/manager/controller_manager_config.yaml#L53?